projects/distributed-tracing/src/shared/dashboard/widgets/span-detail/data/span-detail-data-source.model.ts

```diff
@@ -49,6 +49,7 @@ export class SpanDetailDataSourceModel extends GraphQlDataSourceModel<SpanDetail
   protected constructSpanDetail(span: Span): SpanDetailData {
     return {
       id: span[spanIdKey],
+      traceId: span.apiTraceId as string,
```

Comment:
Should it be apiTraceId here? I remember working on a change where Buchi asked me to show Trace Id instead.

Reply:
apiTraceId is not trace ID - we should be using the real thing here.